packages/app-project/src/screens/ProjectHomePage/components/Hero/components/WorkflowSelector/components/WorkflowSelectButton/theme.js

```diff
@@ -1,20 +1,32 @@
+import { adjustHue } from 'polished'
+
 const theme = {
   button: {
     border: {
       width: '0'
     },
-    color: 'black',
-    extend: props => `
-      text-align: center;
-      background: ${props.theme.global.colors['neutral-4']};
-      &:hover {
-        box-shadow: none;
-      }
-      &:disabled {
-        cursor: not-allowed;
-      }
-    `
-  }
+    color: {
+      dark: 'neutral-4',
+      light: 'black'
+    },
+    extend: props => {
+      const { theme: { dark, global: { colors } } } = props
+      
+      return `
+        text-align: center;
+        background: ${props.theme.global.colors['neutral-4']};
```

Comment:
just noticed, this could also be `background: ${colors['neutral-4']};`

Reply:
Good point.

Reply:
I haven't figured out how to get the dark theme working. Simply copying over the dark settings from `theme.js` for the Next button doesn't quite work.

Reply:
Nevermind, theming has been added now (also copied from the Next button in the classifier.)